tests/chainer_tests/iterators_tests/test_serial_iterator.py

```diff
@@ -205,7 +216,8 @@ def test_iterator_not_repeat_not_even(self):
 
     def test_iterator_shuffle_divisible(self):
         dataset = list(range(10))
-        it = iterators.SerialIterator(dataset, 10)
+        it = iterators.SerialIterator(dataset, 2, shuffle=self.shuffle,
```

Comment:
Any particular reason the minibatch size is changed?

Reply:
Oh. This is my mistake.